cmake/Buildbayes-filters-lib.cmake

```
@@ -0,0 +1,15 @@
+# Copyright (C) Fondazione Istituto Italiano di Tecnologia
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+include(FindOrBuildPackage)
+
+ycm_ep_helper(bayes-filters-lib TYPE GIT
```

```suggestion
ycm_ep_helper(BayesFilters TYPE GIT
```

For historical reason (basically so that `find_or_build_package` works fine if we wanted to find the package outside the superbuild), we use CMake package names as identifiers for packages in the superbuild, so in this context we would need `BayesFilters` (see https://github.com/robotology/bayes-filters-lib/blob/0.10.0/CMakeLists.txt#L12). Note that we also need to change the file name and the calls to `find_or_build_package(bayes-filters-lib)` .